client/src/components/Header.js

```diff
@@ -9,8 +10,9 @@ import MenuIcon from "assets/icons/menu.svg";
 import envelope from "assets/icons/envelope.svg";
 import logo from "assets/logo.svg";
 import Logo from "./Logo";
+
```

Comment:
Remove this extra white space.

Reply:
Maybe addthis import somewhere else, as this section of imports are marked with //icons

Reply:
I added it to logically separate the imports

Reply:
Ohh I see it now. Makes sense :)

Reply:
I could add a // Constants comment but I feel like it's unnecessary to explain imports so explicitly

Reply:
Agee.